### PR TITLE
migrate from JUnit 4 to JUnit 5

### PR DESCRIPTION
**Pull request by sullis**
This migration was accomplished using the OpenRewrite JUnit 5 recipe.

```
activeRecipe("org.openrewrite.java.testing.junit5.JUnit5BestPractices")
```

The recipe was applied by running this Gradle command:

```
./gradlew clean rewriteRun
```

Reference:
https://docs.openrewrite.org

### CODE REVIEW

File: zuul-groovy/src/test/java/com/netflix/zuul/scriptManager/FilterVerifierTest.java [hunk 1]
```diff
@@ -134,21 +131,21 @@ public void testZuulFilterInstance() throws Exception {
...
     }
 
     @Test
-    public void testVerify() throws Exception {
+    void testVerify() throws Exception {
         FilterInfo filterInfo1 = FilterVerifier.INSTANCE.verifyFilter(sGoodGroovyScriptFilter);
         assertNotNull(filterInfo1);
-        assertEquals(filterInfo1.getFilterID(), "null:filter:in");
-        assertEquals(filterInfo1.getFilterType(), FilterType.INBOUND);
-        assertEquals(filterInfo1.getFilterName(), "filter");
```
Comment (gavinbunney) [inline zuul-groovy/src/test/java/com/netflix/zuul/scriptManager/FilterVerifierTest.java:146]: Not sure if this is the formatter playing from the rewrite tool? Maybe worthwhile running these through the internal formatter to keep things consistent

**Review by gavinbunney — APPROVED** [review]
Assuming this works - awesome! One question around formatting otherwise 🚀

**Comment by sullis** [comment]
> Assuming this works - awesome! One question around formatting otherwise 🚀

Fun fact about OpenRewrite:  it was originally created at Netflix.  It was initially published as a Netflix OSS project:
https://github.com/Netflix-Skunkworks/rewrite

The rewrite project was later reborn as OpenRewrite:
https://docs.openrewrite.org

File: zuul-groovy/src/test/java/com/netflix/zuul/scriptManager/FilterVerifierTest.java (hunk 1 above)
Comment (sullis) [reply zuul-groovy/src/test/java/com/netflix/zuul/scriptManager/FilterVerifierTest.java]: I just pushed a fix for FilterVerifierTest.  OpenRewrite should have preserved whitespace in this file, but it did not.

**Pull request merged**